### PR TITLE
[MTM-55237] Export time ranges - LAST_24H, LAST_7DAYS, LAST_30DAYS

### PR DESCRIPTION
**Pull request by pawelkowalczyksag**
Its a bit extensive, maybe would be better to have just examples.

### CODE REVIEW

File: content/cockpit/managing-exports.md [hunk 1]
```diff
@@ -57,7 +57,35 @@ To filter for a particular object, enter a name or property value into the searc
...
+  - To: the time of Export
+  - Example: from 2020-12-31 12:34:56 to 2021-01-01 12:34:56 (when export done at 2021-01-01 12:34:56)
+- Last 7 days:
+  - From: exactly 7 days before the time of Export
+  - To: the time of Export
+  - Example: from 2020-12-25 12:34:56 to 2021-01-01 12:34:56 (when export done at 2021-01-01 12:34:56)
+- Last 30 days:
+  - From: exactly 30 days before the time of Export
+  - To: the time of Export
+  - Example: from 2020-12-02 12:34:56 to 2021-01-01 12:34:56 (when export done at 2021-01-01 12:34:56)
```
Comment (Airdroplet) [inline content/cockpit/managing-exports.md:85]: Maybe would be good to move description of  those options with numbers in names above all other options?

Like in order: 
24
7
30
week
month
year
custom

**Review by Airdroplet — APPROVED** [review]

File: content/cockpit/managing-exports.md [hunk 2]
```diff
@@ -57,7 +57,35 @@ To filter for a particular object, enter a name or property value into the searc
 If you select a group, the data of direct child devices will be included. However the export will not contain the data of devices in subgroups (indirect children).
 {{< /c8y-admon-info >}}
 
-The **Time range** filter can filter data for a specific time range. Select a time range from the dropdown field. This may be one of "Last year", "Last month", "Last week", or you may select "Custom" and enter a custom from/to range in the additional fields.
+The **Time range** filter can filter data for a specific time range. Select a time range from the dropdown field. All times assume platform system time (UTC).
+
+- Last 24 hours:
+  - From: exactly 24 hours before the time of Export
```
Comment (rssag) [inline content/cockpit/managing-exports.md:63]: "export" should be lowercase. multiple occurrences

File: content/cockpit/managing-exports.md [hunk 3]
```diff
@@ -57,7 +57,35 @@ To filter for a particular object, enter a name or property value into the searc
...
+  - To: the time of Export
+  - Example: from 2020-12-31 12:34:56 to 2021-01-01 12:34:56 (when export done at 2021-01-01 12:34:56)
```
Comment (rssag) [inline content/cockpit/managing-exports.md:65]:
```suggestion
  - Example: from 2020-12-31 12:34:56 to 2021-01-01 12:34:56 (when the export is done at 2021-01-01 12:34:56)
```
also multiple occurrences

File: content/cockpit/managing-exports.md [hunk 4]
```diff
@@ -57,7 +57,35 @@ To filter for a particular object, enter a name or property value into the searc
...
+- Last month:
+  - From: same day as the time of Export, but last month at midnight
+  - To: the time of Export
+  - Example: from 2020-12-01 00:00:00 to 2021-01-01 12:34:56 (when export done at 2021-01-01 12:34:56)
+- Last year:
+  - From: same day as the time of Export, but last year at midnight
+  - To: the time of Export
+  - Example: from 2020-01-01 00:00:00 to 2021-01-01 12:34:56 (when export done at 2021-01-01 12:34:56)
+- Custom:
+  - From: date and time user specified
```
Comment (rssag) [inline content/cockpit/managing-exports.md:87]:
```suggestion
  - From: date and time specified by the user
```
also below

**Review by rssag — CHANGES_REQUESTED** [review]

File: content/cockpit/managing-exports.md (hunk 2 above)
Comment (pawelkowalczyksag) [reply content/cockpit/managing-exports.md:63]: Honestly I thought "Export" as a name of the feature should be uppercase. But sure

**Comment by pawelkowalczyksag** [comment]
@rssag hope its okay now

**Review by rssag — APPROVED** [review]

**Comment by rssag** [comment]
what are the target versions?

**Comment by pawelkowalczyksag** [comment]
> what are the target versions?

I do not know yet know but probably only newest. I was asked by customer to backport but they requested outdated version so... :D

**Review by pawel-rynarzewski-sag — APPROVED** [review]

**Review by kaczynskid-sag — APPROVED** [review]

**Review by jkot-softwareag — APPROVED** [review]

**Pull request merged**